prow/config.yaml

```diff
@@ -3039,3 +3143,41 @@ periodics:
     - hostPath:
         path: /mnt/disks/ssd0
       name: cache-ssd
+
+- name: ci-kubernetes-pull-gce-federation-deploy-canary
+  interval: 168h
```

Comment:
I think we should run this every day. I would like this to be as close to our presubmits as possible.